deploy/98_fixtures.ts

```diff
@@ -9,8 +9,6 @@ const func: DeployFunction = async function (env: HardhatRuntimeEnvironment) {
 
   const [owner, alice, bob, treasury] = await ethers.getSigners();
 
-  console.table([alice, bob, treasury]);
```

Comment:
is this intentional? I used these variables sometimes

Reply:
also no. was meant to be reverted